docs/crio.conf.5.md

```diff
@@ -206,6 +206,10 @@ the container runtime configuration.
 **read_only**=false
   If set to true, all containers will run in read-only mode.
 
+**allow_userns_annotation**=false
+  Allow usage of the experimental feature for setting user namespaces through the io.kubernetes.cri-o.userns-mode annotation.  This is an experimental feature,
```

Comment:
nit: maybe we should do it like the crio.8.md file as [EXPERIMENTAL] so it stands out.